src/olympia/templates/photon-footer.html

```diff
@@ -20,24 +20,34 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
             </ul>
         </section>
 
-        <section class="Footer-firefox-links">
-            <h4 class="Footer-links-header"><a href="https://www.mozilla.org/firefox/">Firefox</a></h4>
+        <section class="Footer-browsers-links">
+            <h4 class="Footer-links-header">Browsers</a></h4>
 
             <ul class="Footer-links">
-                <li><a href="https://www.mozilla.org/firefox/new/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Download Firefox</a></li>
-                <li><a href="https://www.mozilla.org/firefox/android/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Android Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/ios/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">iOS Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/focus/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Focus Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/desktop/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Desktop Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/channel/desktop/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Beta, Nightly, Developer Edition</a></li>
+                <li><a href="https://www.mozilla.org/firefox/new/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Desktop</a></li>
+                <li><a href="https://www.mozilla.org/firefox/mobile/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Mobile</a></li>
+                <li><a href="https://mixedreality.mozilla.org/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link">Reality</a></li>
+                <li><a href="https://www.mozilla.org/firefox/enterprise/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Enterprise</a></li>
             </ul>
+        </section>
+
+        <section class="Footer-product-links">
+            <h4 class="Footer-links-header">Products</h4>
+
+            <ul class="Footer-links">
+                <li><a href="https://www.mozilla.org/firefox/lockwise/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Lockwise</a></li>
```

Comment:
My comments above apply here as well. I.e., we should probably use `&amp;`, and the params should be in the same order for each link.

Reply:
Most of these changes were inspired from the format at https://addons.mozilla.org/en-US/firefox/ , I guess once this resolve we should be fixing those too